maven/lib/dependabot/maven/update_checker/version_finder.rb

```diff
@@ -140,18 +140,15 @@ def released?(version)
           @released_check[version] =
             repositories.any? do |repository_details|
               url = repository_details.fetch("url")
-              response = Excon.get(
+              response = Excon.head(
```

Comment:
Oh looks like this request needs to be stubbed out in a bunch of tests, looks like this might not be that bad, just need to check for places where we already stub out the old request url and change it to include the `jar` file, example in the update checker: https://github.com/dependabot/dependabot-core/blob/master/maven/spec/dependabot/maven/update_checker_spec.rb#L58-L61

Reply:
I started fixing this, but I'm unsure as to why the example on line 86 is still failing.  I changed the`stub_request` to be `:head` and I added `metadata` to `:dependency_requirements`.